app/views/questionnaire.py

```diff
@@ -285,9 +287,9 @@ def get_view_submission(eq_id, form_type):  # pylint: disable=unused-argument
     return redirect(url_for('post_submission.get_thank_you', eq_id=eq_id, form_type=form_type))
 
 
-def _redirect_to_latest_location(routing_path, collection_id, eq_id, form_type):
-    latest_location = path_finder.get_latest_location(get_completed_blocks(current_user),
-                                                      routing_path=routing_path)
+def _redirect_to_latest_location(collection_id, eq_id, form_type):
+    latest_location = completeness.get_latest_location()
```

Comment:
I think if we make `get_completeness` a wrapper then it would be more obvious where `completeness` is coming from.

Reply:
I did that for a reason... but I can't remember what it was now. I'll find out and clarify.